rossum_api/elis_api_client_sync.py

```diff
@@ -126,7 +126,7 @@ def import_document(
     ) -> List[int]:
         """https://elis.rossum.ai/api/docs/#import-a-document.
 
-        Deprecated now, consider create_upload.
+        Deprecated now, consider upload_document.
```

Comment:
Commit separation isn't ideal in this MR, I've seen you fixing issues introduced in one commit in a later one instead of amending the change but I don't want to force you clean it up now. Just something to be careful about, it's not great for code review commit by commit -- I wrote some comments about problems I've noticed only to discover that you fixed it later.

Reply:
Yeah, sorry about that. Looking into re-organizing it.